oauthproxy_test.go

```diff
@@ -164,7 +164,7 @@ func TestRobotsTxt(t *testing.T) {
 	opts := NewOptions()
 	opts.ClientID = "asdlkjx"
 	opts.ClientSecret = "alkgks"
-	opts.Cookie.Secret = "asdkugkj"
+	opts.Cookie.Secret = "secretthirtytwobytes+abcdefghijk"
```

Comment:
Looks like we aren't showing examples of using a base64 encoded 32 bit secret. Might be nice from a tests as documentation perspective. Also helps with coverage.